Classes/Console/Command/Documentation/GenerateXsdCommand.php

```diff
@@ -0,0 +1,125 @@
+<?php
+declare(strict_types=1);
+namespace Helhum\Typo3Console\Command\Documentation;
+
+/*
+ * This file is part of the TYPO3 Console project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read
+ * LICENSE file that was distributed with this source code.
+ *
+ */
+
+use Helhum\Typo3Console\Command\AbstractConvertedCommand;
+use Helhum\Typo3Console\Service;
+use Helhum\Typo3Console\Service\XsdGenerator;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class GenerateXsdCommand extends AbstractConvertedCommand
+{
+    /**
+     * @var XsdGenerator
+     */
+    protected $xsdGenerator;
+
+    public function __construct(string $name = null, XsdGenerator $xsdGenerator= null)
+    {
+        parent::__construct($name);
+        $this->xsdGenerator = $xsdGenerator;
+    }
+
+    protected function configure()
+    {
+        $this->setDescription('Generate Fluid ViewHelper XSD Schema');
+        $this->setHelp(
+            <<<'EOH'
+Generates Schema documentation (XSD) for your ViewHelpers, preparing the
+file to be placed online and used by any XSD-aware editor.
+After creating the XSD file, reference it in your IDE and import the namespace
+in your Fluid template by adding the xmlns:* attribute(s):
+<code><html xmlns="http://www.w3.org/1999/xhtml" xmlns:f="http://typo3.org/ns/TYPO3/Fluid/ViewHelpers" ...></code>
+EOH
+        );
+        $this->setDefinition($this->createCompleteInputDefinition());
+    }
+
+    protected function createNativeDefinition(): array
+    {
+        return [
+            new InputArgument(
+                'phpNamespace',
+                InputArgument::REQUIRED,
+                'Namespace of the Fluid ViewHelpers without leading backslash (for example \'TYPO3\Fluid\ViewHelpers\' or \'Tx_News_ViewHelpers\'). NOTE: Quote and/or escape this argument as needed to avoid backslashes from being interpreted!'
+            ),
+            new InputOption(
+                'xsd-namespace',
+                '-xsd',
+                InputOption::VALUE_REQUIRED,
+                'Unique target namespace used in the XSD schema (for example "http://yourdomain.org/ns/viewhelpers"). Defaults to "http://typo3.org/ns/<php namespace>".'
+            ),
+            new InputOption(
+                'target-file',
+                '-target',
```

Comment:
Maybe make this `-t` for a real short option.

Reply:
done